gitleaks.toml

```diff
@@ -104,3 +104,9 @@ title = "gitleaks config"
 	files = ['''^\.?gitleaks.toml$''',
 	'''(.*?)(jpg|gif|doc|pdf|bin)$''',
 	'''(go.mod|go.sum)$''']
+	commits = [
```

Comment:
For what purpose will the commits be put on the allow list?

Reply:
Those commits contain test secrets. Probably need a comment here. Will add.